web/templates/input_pill.hbs

```diff
@@ -3,7 +3,7 @@
     <img class="pill-image" src="{{img_src}}" />
     {{/if}}
     <span class="pill-label">
-        <span class="pill-value">{{ display_value }}</span>
+        <span class="pill-value">{{#if deactivated}}{{{display_value}}}{{else}}{{display_value}}{{/if}}</span>
```

Comment:
We try to avoid the triple `{{{` whenever possible, as it presents a tricky security risk. A better approach here would probably be to create a separate handlebars template for deactivated users (or perhaps all users), and render that just like the `status_emoji_info` template in the lines below.